x-pack/plugins/case/server/client/cases/mock.ts

```diff
@@ -76,6 +77,14 @@ export const commentAlert: CommentResponse = {
   version: 'WzEsMV0=',
 };
 
+export const commentAlertMultipleIds: CommentResponseAlertsType = {
+  ...commentAlert,
+  id: 'mock-comment-2',
+  alertId: ['alert-id-1', 'alert-id-2'],
+  index: 'alert-index-1',
+  type: CommentType.alert as const,
```

Comment:
This is cool, didn't realize this added more type safety. Just for my own edification, what does this enforce for `type`?

Reply:
@jonathan-buttner This enforce the variable to be of type `"alert"` and not `string`. If you do `const x = "test"` the type of `x` is `string`. If you do `const x = "test" as const` the type of `x` is `"test"`, a constant.

You can read more here: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions